slideshow/src/main/java/nl/ou/desktop/ContentRenderer.java

```diff
@@ -35,15 +34,27 @@ public void doForFigure(Figure figure) {
             throw new RuntimeException(ex);
         }
     }
-    
+
     @Override
     public void doForListStart(ListContent content) {
-        final var layout = new GridLayout(0, 1);
-        final var list = new JPanel(layout);
-        current.add(list);     
+        int level = content.getLevel();
```

Comment:
Zou je niet indentation level in de visitor kunnen berekenen door op het eind van doForListStart met 1 te incrementen en bij doForListEnd te decrementen? Dan hoef je niks aan te passen in het domein. Dat level kun je dan opslaan in deze class en die zou op het einde van de visitor calls natuurlijk weer 0 moeten zijn.

Reply:
Ja en nee. Nee, want e moet hier rekening houden met een list in een tabel, waarbij je qua level weer bij 0 wilt beginnen. Ja, want er is een veel simpeler oplossing om dat probleem te fixen.

Nieuw PR daarvoor aangemaakt, zie https://github.com/Borrelhapje/design-for-change/pull/17